generated/java/google-cloud-speech-v1beta1/src/main/java/com/google/cloud/speech/spi/v1beta1/SpeechApi.java

```diff
@@ -146,6 +154,16 @@ public void close() throws IOException {
     }
   }
 
+  public final StreamObserver<StreamingRecognizeRequest> streamingRecognize(
+      StreamObserver<StreamingRecognizeResponse> streamObserver) {
+    return streamingRecognizeCallable.call(streamObserver);
+  }
+
+  public final ApiCallable<StreamObserver<StreamingRecognizeResponse>,
+      StreamObserver<StreamingRecognizeRequest>> streamingRecognizeCallable() {
+    return streamingRecognizeCallable;
+  }
```

Comment:
I'm wondering if we should go fully minimalist with this and only provide the callable.